### ISSUE TITLE
Full text extraction contrib libraries as option

### ISSUE THREAD

**Issue by grosscol** [projecthydra/jettywrapper#46]
Some hydra development relies on additional functionality in solr. In order to support that development, the projects have been pulling in the libraries and modifying the solrconfig.xml using an install generator.  With this functionality being pushed down the stack to hydra works projecthydra-labs/hydra-works#195, the option to use an install generator goes away.

Is adding contrib libraries and modifying the solrconfig something that the community is interested in providing as an option at the jettywrapper level?  This could be related to #33, as optional arguments could be a method of implementing this.

**Comment by jcoyne** [comment]
Sure. So you are proposing something like a rake task?  rake jetty:install:fulltext or something?

**Comment by awead** [comment]
There's already this set of tasks in Sufia that enable fulltext indexing:

https://github.com/projecthydra/sufia/blob/master/sufia-models/lib/tasks/sufia-models_tasks.rake#L37

It just downloads the appropriate jars. There's are solr config files in Sufia as well that are copied over to the jetty directory as part of this process too. It would be nice if different config options could be exacted and shared across different projects. Maybe jettywrapper is the place to do that.

**Comment by grosscol** [comment]
@jcoyne, Yes.  A rake task to do install fulltext or something similar would be convenient.  Since it's a jettywrapper related configuration change, the thought was that it should be an optional task that lives in jetty.
